src/Generation/FieldDetails.php

```diff
@@ -138,15 +138,16 @@ class FieldDetails
      * AFTER a package's 1.0 release, back to being optional.
      */
     private static $optionalToRequiredFixes = [
-        'google.logging.v2.UpdateCmekSettingsRequest' => ['name', 'cmek_settings'],
-        'google.logging.v2.GetCmekSettingsRequest' => ['name'],
+        'google.cloud.clouddms.v1.DescribeDatabaseEntitiesRequest' => ['tree'],
```

Comment:
What is the `tree` part and where is it used?

Reply:
I don't know anything about it other than it's a property that used to be optional and now it's required.